pack/fhv/fhvc.json

```diff
@@ -2904,6 +2904,106 @@
+    "type_code": "enemy"
+  },
+  {
+    "code": "10733",
+    "encounter_code": "the_forest",
+    "encounter_position": 3,
+    "enemy_damage": 1,
+    "enemy_evade": 3,
+    "enemy_fight": 1,
+    "faction_code": "mythos",
+    "flavor": "Don't even try to pet it.",
+    "health": 2,
+    "illustrator": "Carlos Palma Cruchaga",
+    "name": "Forest Watcher",
+    "pack_code": "fhvc",
+    "position": 233,
+    "quantity": 2,
+    "text": "Aloof. <b>(Night [night])</b> Elusive.\n<b>Spawn</b> – Any location (empty, if able).\n<b>Forced</b> – After Forest Watcher enters play: Place 1 doom on it.",
+    "traits": "Creature. Flora. Mutated.",
+    "type_code": "enemy"
+  },
+  {
+    "code": "10734",
+    "encounter_code": "the_forest",
+    "encounter_position": 5,
+    "enemy_damage": 2,
+    "enemy_evade": 4,
+    "enemy_horror": 1,
+    "enemy_fight": 3,
+    "faction_code": "mythos",
+    "flavor": "Half the majestic creature's flesh has been replaced with vascular, bark-like tissue.",
+    "health": 5,
+    "illustrator": "Peter Polach",
+    "name": "Cochleal Stag",
+    "pack_code": "fhvc",
+    "position": 234,
+    "quantity": 1,
+    "text": "Elusive. <b>(Night [night])</b> Hunter.\nEach investigator at Cochleal Stag's location and each connecting location gets -2 to all skills during the mythos phase.",
+    "traits": "Creature. Flora. Mutated.",
+    "type_code": "enemy"
+  },
+  {
+    "code": "10735",
+    "encounter_code": "the_forest",
+    "encounter_position": 6,
+    "faction_code": "mythos",
+    "illustrator": "Rob Laskey",
+    "name": "Bloom",
+    "pack_code": "fhvc",
+    "position": 235,
+    "quantity": 2,
+    "text": "<b>Revelation</b> - Choose the nearest [[Flora]] enemy and heal all damage from it (if that enemy is [[Elite]], heal 2[per_investigator] damage from it instead). If no damage was healed by this effect, discard cards from the top of the encounter deck until a [[Flora]] card is discarded and draw it.",
+    "traits": "Power.",
+    "type_code": "treachery"
+  },
+  {
+    "code": "10736",
+    "encounter_code": "the_forest",
+    "encounter_position": 8,
+    "faction_code": "mythos",
+    "illustrator": "Carlos Palma Cruchaga",
+    "name": "Wall of Thorns",
+    "pack_code": "fhvc",
+    "position": 236,
+    "quantity": 2,
+    "text": "<b>Revelation</b> - Attach Wall of Thorns to the nearest location without Wall of Thorns attached.\n<b>(Day [day]) Forced</b> - When you move into attached location: Take 2 damage. Discard Wall of Thorns.\n<b>(Night [night]) Forced</b> - When you move into or out of attached location: Take 1 damage.",
+    "traits": "Hazard. Flora.",
+    "type_code": "treachery"
+  },
+  {
+    "code": "10737",
+    "encounter_code": "the_forest",
+    "encounter_position": 10,
+    "faction_code": "mythos",
+    "flavor": "\"...even the herbivores seem to have developed an unusual ability to plan, coordinate, and hunt.\"\n - Dr. Rosa Marquez, Miskatonic Survey, 1926",
```

Comment:
```suggestion
    "flavor": "\"...even the herbivores seem to have developed an unusual ability to plan, coordinate, and hunt.\"\n - Dr. Rosa Marquez, <u>Miskatonic Survey, 1926</u>",
```

Reply:
We pop a wee underline under italicised titles in flavor text, just to show they are a book title.